### ISSUE TITLE
"Tool" template doesn't have any entitlements

### ISSUE THREAD

**Issue by Muirey03** [theos/templates#11]
On iOS 11+, "tools" are required to have these entitlements otherwise they will crash with `Killed: 9`:
```
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>platform-application</key>
    <true/>
    <key>com.apple.private.security.container-required</key>
    <false/>
</dict>
</plist>
```
Please could you put these entitlements into an ent.xml, and add them to the tool template (setting `CODESIGN_FLAGS` of course) so that new tools don't immediately crash when you try to run them?

**Comment by andermoran** [comment]
For the iOS 12 Chimera jailbreak, the lines needed for entitlements are:
```
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>platform-application</key>
    <true/>
</dict>
</plist>
```
If the following key is added,
```
    <key>com.apple.private.security.container-required</key>
    <false/>
```
the tool will output `Killed 9`

**Comment by uroboro** [comment]
Fixed in 7d98646871281e5b71e280e8f80d3fc27b69aeef
